### PR TITLE
handle tier0 test setting

### PR DESCRIPTION
**Pull request by ticoann**
@hufnagel, Dirk please review

### CODE REVIEW

File: bin/wmagent-couchapp-init [hunk 1]
```diff
@@ -367,6 +367,10 @@ if __name__ == "__main__":
     
     if hasattr(wmagentConfig, "Tier0Feeder"):
         installCouchApp(wmagentConfig.JobStateMachine.couchurl, wmagentConfig.Tier0Feeder.requestDBName, "T0Request")
+        centralWMStatsURL = config.AgentStatusWatcher.centralWMStatsURL
```
Comment (hufnagel) [inline bin/wmagent-couchapp-init]: Copy&Paste error ? This should be wmagentConfig.AgentStatusWatcher.centralWMStatsURL, right ?

Comment (ticoann) [reply bin/wmagent-couchapp-init]: Thanks. updated now

File: bin/wmagent-couchapp-init [hunk 2]
```diff
@@ -367,6 +367,11 @@ if __name__ == "__main__":
     
     if hasattr(wmagentConfig, "Tier0Feeder"):
         installCouchApp(wmagentConfig.JobStateMachine.couchurl, wmagentConfig.Tier0Feeder.requestDBName, "T0Request")
+        centralWMStatsURL = wmagentConfig.AgentStatusWatcher.centralWMStatsURL
+        # checks whether this is test agent. if T0 agent is setting local wmstats as central it creates the db
+        if "localhost:" in centralWMStatsURL:
+            wmstatsDBName = a.split("/")[-1].strip()
```
Comment (hufnagel) [inline bin/wmagent-couchapp-init]: centralWMStatsURL.split ;-) ?

Comment (ticoann) [reply bin/wmagent-couchapp-init]: Sorry another copy and paste error. updated

**Comment by hufnagel** [comment]
Ok, tested in a replay, works great.

**Comment by ticoann** [comment]
Thanks Dirk, I will include in wmagent branch.

**Pull request merged**